### ISSUE TITLE
bug: ` settings file is not a json object`

### ISSUE THREAD

**Issue by Stubbs** [withfig/fig#1987]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.0 (21G115)

### Expected behaviour

I want to open the Fig dashboard from the menu at the top of the screen.

### Actual behaviour

I'm trying to open the Fig dashboard, after logging in I get the following error message:

```
Something went wrong :(
Error: Failed setting previous_email: settings file is not a json object
```

If I click try again I get the same error or this error

`Error: Failed setting user_id: settings file is not a json object`

and it alternates between the 2.

```
~ ᐅ fig doctor

✘ PATH and PseudoTerminal PATH match: Could not get PseudoTerminal PATH
  settings file is not a json object


✘ Doctor found errors. Please fix them and try again.
```

I restarted & get the same error.

### Steps to reproduce

See above.

### Environment

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model:
  - model-id:
  - chip-id: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  - cores: 6
  - mem: 32.00 GB
os-info:
  - macOS 12.6.0 (21G115)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/stuart
  - exe-path: /usr/local/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: c63a0ceb-d5d3-42ff-9a49-ebb28df5fe40
    - FIG_HOSTNAME: MacBook-Pro-2.local
    - FIG_TERM: 2.7.4
    - PATH: /usr/local/Cellar/zplug/2.4.2/bin:/Users/stuart/.nvm/versions/node/v18.11.0/bin:/Users/stuart/.rbenv/shims:/usr/local/opt/mysql-client/bin:/usr/local/sbin:/usr/local/sahi/bin:/Users/stuart/.composer/vendor/bin:/Users/stuart/Projects/git-achievements:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/X11/bin:/Users/stuart/bin:/usr/local/go/bin:/Users/stuart/Projects/spark-installer:/Users/stuart/.rvm/bin:/Users/stuart/Library/Python/3.7/bin:/Users/stuart/Projects/flutter/bin:/Users/stuart/.composer/vendor/bin:/Users/stuart/.composer/vendor/bin:/Users/stuart/.fig/bin:/Users/stuart/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm
    - FIG_PID: 1571
    - FIG_PARENT: c63a0ceb-d5d3-42ff-9a49-ebb28df5fe40
    - FIG_SET_PARENT: c63a0ceb-d5d3-42ff-9a49-ebb28df5fe40
```

**Comment by robbiedavey** [comment]
Getting the same error just now.

**Comment by grant0417** [comment]
If this is happening to you try running `rm ~/Library/Application\ Support/fig/state.json`

**Comment by grant0417** [comment]
This should be fixed on 2.7.7, make sure to update and let us know if the issue is fixed!
